add_to_app/flutter_module_books/test/widget_test.dart

```diff
@@ -0,0 +1,30 @@
+// This is a basic Flutter widget test.
```

Comment:
This test is broken, right? Is there a way to replace it with a working test?

I ask because the add-to-app apps and modules are currently all integrated into CI. In the `/tool` directory there's a script for pure Flutter stuff and two more for Android and iOS. This should be wired into those scripts before it's landed.

Reply:
You don't have to go as far as the espresso tests that are in one of the Android apps, I should mention.